### PR TITLE
Add osm related URL in whitelist for monlogis app.

### PR DESCRIPTION
**Pull request by jacquarg**
This PR add OSM nominatin API and http://osmbuildings.org/ in whitelist, for the app monlogis  : https://github.com/jacquarg/monlogis.

### CODE REVIEW

File: web/routing.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ func newRenderer(assetsPath string) (*renderer, error) {
 func SetupAppsHandler(appsHandler echo.HandlerFunc) echo.HandlerFunc {
 	secure := middlewares.Secure(&middlewares.SecureConfig{
 		HSTSMaxAge:    hstsMaxAge,
-		CSPDefaultSrc: []middlewares.CSPSource{middlewares.CSPSrcSelf, middlewares.CSPSrcParent},
+		CSPDefaultSrc: []middlewares.CSPSource{middlewares.CSPSrcSelf, middlewares.CSPSrcParent, middlewares.CSPSrcWhitelist},
```
Comment (nono) [inline web/routing.go]: You also need the whitelist on default for this PR?

Comment (jacquarg) [inline web/routing.go]: I not totally certain, but they are not images at least, both are json APIs :

nominatim.openstreetmap.org :
```
curl 'http://nominatim.openstreetmap.org/search?format=json&q=2%20BOULEVARD%20DES%20BELGES+LYON+FRANCE' -H 'Host: nominatim.openstreetmap.org' -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0' -H 'Accept: */*' -H 'Accept-Language: fr,fr-FR;q=0.8,en-US;q=0.5,en;q=0.3' --compressed -H 'Referer: http://app.cozy.tools:8080/' -H 'Origin: http://app.cozy.tools:8080' -H 'Connection: keep-alive' -H 'Cache-Control: max-age=0'
```

osmbuildings.org : 
```
curl 'http://c.data.osmbuildings.org/0.2/anonymous/tile/15/16824/11686.json' -H 'Host: c.data.osmbuildings.org' -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0' -H 'Accept: */*' -H 'Accept-Language: fr,fr-FR;q=0.8,en-US;q=0.5,en;q=0.3' --compressed -H 'Referer: http://app.cozy.tools:8080/' -H 'Origin: http://app.cozy.tools:8080' -H 'Connection: keep-alive'
```

Comment (nono) [inline web/routing.go]: Well, I'm annoyed.

When we started the whitelist, we have said that it was OK to load images (and maybe other assets, sounds? videos?) from other domains, but we don't want scripts and even less JSON APIs or websockets. But our proxy for these stuff is not here, and you have several apps that need it.

@jinroh, @aenario, @poupotte: what do you think about that?

**Comment by nono** [comment]
Hi @jacquarg, what is the status on this PR?

**Comment by nono** [comment]
@jacquarg I close this pull request, but feel free to reopen it if you want to discuss about it. We may merge it if the remote doctypes don't work for this use case.

**Pull request closed without merging**